config/jobs/kubernetes/sig-release/cip/container-image-promoter.yaml

```diff
@@ -0,0 +1,29 @@
+presubmits:
+  kubernetes/k8s.io:
+  - name: cip-dryrun
+    decorate: true
+    skip_report: false
+    run_if_changed: ".*/manifest.yaml"
+    # It's OK to run things concurrently, because we're using "-dry-run".
+    max_concurrency: 10
+    branches:
+    - ^master$
+    spec:
+      containers:
+      - image: gcr.io/cip-demo-staging/cip@sha256:876bfc74d070f2eeb94625780b78e757834e1854ec30bc539cfb7553274905d9
+        command:
+        - multirun.sh
+        args:
+        - /app/cip-docker-image.binary
+        - k8s-staging-cluster-api/manifest.yaml
+        - k8s-staging-coredns/manifest.yaml
+        - k8s-staging-csi/manifest.yaml
+        env:
+        - name: CIP_OPTS
```

Comment:
Can invert this so that it runs dry by default and the prod jobs set `--dry-run=false`? This seems like the safer, more appropriate default